examples/minicpmv/requirements.txt

```diff
@@ -0,0 +1,3 @@
+-r ../../requirements/requirements-convert.txt
+pillow~=10.2.0
+torch~=2.1.1
```

Comment:
it's missing `torchvision` required by your `modeling_minicpmv.py` from huggingface

Reply:
> it's missing `torchvision` required by your `modeling_minicpmv.py` from huggingface

yes, It's my fault.